src/components/InstancesList/InstancesList.js

```diff
@@ -1150,15 +1122,16 @@ class InstancesList extends React.Component {
     const visibleColumns = this.getVisibleColumns();
     const columnMapping = this.getColumnMapping();
 
-    const onChangeIndex = (e) => {
+    const onChangeIndex = (e, { isAdvancedSearchModal } = {}) => {
       const qindex = e.target.value;
 
-      this.setState({ optionSelected: qindex });
-
-      parentMutator.query.update({
-        filters: '',
-        ...this.extraParamsToReset,
-      });
+      if (!isAdvancedSearchModal) {
+        parentMutator.query.update({
+          qindex,
```

Comment:
Adding `qindex` is going to break functionality in #2177 where if you search by one option, change dropdown value and switch to Holdings and back - results should include search by option that was used in search

Reply:
#2177 broke this [scenario](https://github.com/folio-org/ui-inventory/assets/77053927/772f807d-5d37-4fca-920d-027560a87546). 
I removed `qindex` because it doesn't relate to this ticket, I thought to fix them together. Will ask QA to create a bug.